`crc status -w` during start can result in a panic of the command

when you do:

```
$ ./crc status -w
```

and start the cluster in another terminal, the command might fail:

```
./crc status -w                                           1 
CRC VM:          Running
MicroShift:      Unreachable (v4.13.3)
Disk Usage:      0B of 0B (Inside the CRC VM)
Cache Usage:     35.05GB
Cache Directory: /home/gbraad/.crc/cache
RAM: -1 B ?%  [██████████████████████████████████████████████████████████████████████████████████████████████████████] 
panic: runtime error: index out of range [0] with length 0

goroutine 1 [running]:
github.com/crc-org/crc/cmd/crc/cmd.runWatchStatus({0x23bb7c0, 0xc000014018}, 0xc00090bb60, {0xc000056cf0?, 0xc0009185b8?})
	/builddir/build/BUILD/crc-2.22.1/cmd/crc/cmd/status.go:104 +0x2a5
github.com/crc-org/crc/cmd/crc/cmd.runStatus({0x23bb7c0?, 0xc000014018?}, 0xc0001e72d0?, {0xc000056cf0?, 0x0?}, {0x0?, 0xc00081fc38?}, 0xd7?)
	/builddir/build/BUILD/crc-2.22.1/cmd/crc/cmd/status.go:59 +0x36
github.com/crc-org/crc/cmd/crc/cmd.glob..func12(0xc0000d6000?, {0x5?, 0x2041ebf?, 0xc?})
	/builddir/build/BUILD/crc-2.22.1/cmd/crc/cmd/status.go:37 +0x53
github.com/crc-org/crc/cmd/crc/cmd.executeWithLogging.func1(0x35a50c0, {0xc000940650, 0x0, 0x1})
	/builddir/build/BUILD/crc-2.22.1/cmd/crc/cmd/root.go:170 +0x102
github.com/spf13/cobra.(*Command).execute(0x35a50c0, {0xc000940630, 0x1, 0x1})
	/builddir/build/BUILD/crc-2.22.1/vendor/github.com/spf13/cobra/command.go:916 +0x862
github.com/spf13/cobra.(*Command).ExecuteC(0x35a67c0)
	/builddir/build/BUILD/crc-2.22.1/vendor/github.com/spf13/cobra/command.go:1044 +0x3bd
github.com/spf13/cobra.(*Command).Execute(...)
	/builddir/build/BUILD/crc-2.22.1/vendor/github.com/spf13/cobra/command.go:968
github.com/spf13/cobra.(*Command).ExecuteContext(...)
	/builddir/build/BUILD/crc-2.22.1/vendor/github.com/spf13/cobra/command.go:961
github.com/crc-org/crc/cmd/crc/cmd.Execute()
	/builddir/build/BUILD/crc-2.22.1/cmd/crc/cmd/root.go:105 +0xd9
main.main()
	/builddir/build/BUILD/crc-2.22.1/cmd/crc/main.go:8 +0x17
```

![Screenshot from 2023-06-29 13-59-49](https://github.com/crc-org/crc/assets/1894/90e4454a-9ea5-4c37-99e4-df94a7bfb7af)

--- a/cmd/crc/cmd/status.go
+++ b/cmd/crc/cmd/status.go
@@ -90,6 +90,14 @@ func runWatchStatus(writer io.Writer, client *daemonclient.Client, cacheDir stri
 			if err != nil {
 				return
 			}
+		} else if len(loadResult.CPUUse) > len(cpuBars) {
+			newCPUCount := len(loadResult.CPUUse) - len(cpuBars)
+			oldCPUCount := len(cpuBars)
+			for i := 0; i < newCPUCount; i++ {
+				bar := createCPUBar(oldCPUCount+i, writer)
+				barPull.Add(bar)
+				cpuBars = append(cpuBars, bar)
+			}
 		}
 
 		ramBar.SetTotal(loadResult.RAMSize)
@@ -111,17 +119,21 @@ func createBars(cpuUse []int64, writer io.Writer) (ramBar *pb.ProgressBar, cpuBa
 	ramBar.SetTemplateString(tmpl)
 
 	for i := range cpuUse {
-		bar := pb.New(101)
-		bar.SetWriter(writer)
-		bar.Set(pb.Static, true)
-		tmpl := fmt.Sprintf(`{{ green "CPU%d:" }} {{percent .}} {{string . "my_green_string" | green}} {{ bar . "[" "\u2588" "\u2588" " " "]"}}`, i)
-		bar.SetTemplateString(tmpl)
-		bar.SetMaxWidth(150)
-
+		bar := createCPUBar(i, writer)
 		cpuBars = append(cpuBars, bar)
 	}
 
 	return ramBar, cpuBars
+}
+
+func createCPUBar(cpuNum int, writer io.Writer) *pb.ProgressBar {
+	bar := pb.New(101)
+	bar.SetWriter(writer)
+	bar.Set(pb.Static, true)
+	tmpl := fmt.Sprintf(`{{ green "CPU%d:" }} {{percent .}} {{string . "my_green_string" | green}} {{ bar . "[" "\u2588" "\u2588" " " "]"}}`, cpuNum)
+	bar.SetTemplateString(tmpl)
+	bar.SetMaxWidth(150)
+	return bar
 }
 
 func getStatus(client *daemonclient.Client, cacheDir string) *status {

